frontend/src/concepts/areas/const.ts

```diff
@@ -10,6 +10,9 @@ export const SupportedAreasStateMap: SupportedAreasState = {
   [SupportedArea.CLUSTER_SETTINGS]: {
     featureFlags: ['disableClusterManager'],
   },
+  [SupportedArea.MODEL_REGISTRY_SETTINGS]: {
+    featureFlags: ['disableModelRegistryManager'],
```

Comment:
This should have ate least:

```typescript
    requiredComponents: [StackComponent.MODEL_REGISTRY],
```

and 

```typescript
    requiredCapabilities: [StackCapability.SERVICE_MESH, StackCapability.SERVICE_MESH_AUTHZ],
```

(this is just cause we need to use it with authorino)